src/webnn_native/ops/Gru.cpp

```diff
@@ -41,6 +41,9 @@ namespace webnn_native { namespace op {
             if (options->initialHiddenState != nullptr) {
                 mInputs.push_back(options->initialHiddenState);
             }
+            if (options->returnSequence) {
```

Comment:
Does the first output set the rank to 3? and add some annotation about the number following below W3C Spec.

> The first element of the sequence is a 3-D tensor of shape [num_directions, batch_size, hidden_size], the cell output from the last time step of the network. Additionally, if returnSequence is set to true, the second element is the 4-D output tensor of shape [steps, num_directions, batch_size, hidden_size] containing every cell outputs from each time step in the temporal sequence.